Core/Logic/routers/user_routes.py

```diff
@@ -1,9 +1,105 @@
-from fastapi import APIRouter,Request
+from fastapi import APIRouter, Request
+from pydantic.main import BaseModel
+from pydantic.types import UUID4
+from starlette.status import HTTP_200_OK, HTTP_201_CREATED, HTTP_404_NOT_FOUND
+from pydantic import EmailStr
 
 user_router = APIRouter()
 
 
+class PydanticUserModel(BaseModel):
+    password: str
+    username: str
+    first_name: str
+    last_name: str
+    email: EmailStr
+    Reg_no: str
 
-@user_router.get("/users/me")
-async def read_users_me(request: Request):
+
+class DeleteUserModel(BaseModel):
+    Reg_no: str
+
+
+@user_router.get("/users/current-user")
+async def current_user(request: Request):
     return request.user
+
+
+@user_router.get("/all-users")
+def show_all_users(request: Request):
+    from Logic.models import UserModel
+
+    if request.user.is_authenticated:
+        return list(UserModel.objects.values("Reg_no"))
+
+
+@user_router.post("/user")
```

Comment:
can we have a separate routes and controllers method?
Let the directory structure be like this:
- routes
    - users.py  -> all the routes in the file should be defined with the prefix 'users'. in node theres an option to configure only once. I am not sure if its there in fast-api please check it once. 
- controllers
   - users.py -> here you define all the methods. 

The reason behing doing this it will have a good index access and when you implement different call methods (GET/ POST / PUT) for the same route it will be easier to manage.

Reply:
Yeah makes sense, this implementation may save a lot of headaches later on.

Reply:
Also with regards to the configure only once, are you referring to the packages in node?